### DISCUSSION TITLE
Broken cards swipe demos - Has a breaking change been introduced?

### DISCUSSION THREAD

**Discussion by laurentlahmy** [pmndrs/use-gesture#478] [Q&A]
It seems to me that [the swipe cards example](https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/cards-stack
), referenced [in the docs as example](https://use-gesture.netlify.app/docs/examples/), is broken.

The flick doesn't happen, likely because the condition `!active && trigger` on line 36 never happens.

[An older version of this same example](https://codesandbox.io/embed/j0y0vpz59) (referenced from  https://react-spring.io/) using react-use-gesture seems to work fine.

By the way, here are the updated image links: 
```
const cards = [
    'https://upload.wikimedia.org/wikipedia/commons/f/f5/RWS_Tarot_08_Strength.jpg',
    'https://upload.wikimedia.org/wikipedia/commons/5/53/RWS_Tarot_16_Tower.jpg',
    'https://upload.wikimedia.org/wikipedia/commons/9/9b/RWS_Tarot_07_Chariot.jpg',
    'https://upload.wikimedia.org/wikipedia/commons/3/3a/TheLovers.jpg',
    'https://upload.wikimedia.org/wikipedia/commons/thumb/8/88/RWS_Tarot_02_High_Priestess.jpg/690px-RWS_Tarot_02_High_Priestess.jpg',
    `https://upload.wikimedia.org/wikipedia/commons/d/de/RWS_Tarot_01_Magician.jpg`,
]
```

Possibly related: it also seems to me that [the swipe example in the docs](https://use-gesture.netlify.app/docs/state/#swipe-drag-only) is broken (no swipe happening for me).

**Comment by dbismut** [comment]
No breaking change, but there could be collateral bugs introduced by latest releases. However, besides the images that don't show on the cards (thanks for the updated links), the examples seem to work fine on my iPhone 11 Pro and MacBook Pro. 

What device are you using?

**Reply by laurentlahmy** [reply]
You are correct, everything works as intended, I realized using [three finger drag on macos](https://support.apple.com/en-gb/HT204609) "breaks" the [flick](https://codesandbox.io/s/github/pmndrs/use-gesture/tree/main/demo/src/sandboxes/cards-stack) or [swipe](https://support.apple.com/en-gb/HT204609). Without three finger drag everything works fine. But I understand this may be a low priority edge case. Thanks for your work!

**Reply by dbismut** [reply]
Oh I see. Indeed trackpad swipe isn't supported, but if you have a way to implement that I'd be happy to have a look!
